arch/arm/src/imxrt/imxrt_flexcan.h

```diff
@@ -0,0 +1,125 @@
+/************************************************************************************
+ * arch/arm/src/imxrt/imxrt_flexcan.h
+ *
+ *   Copyright (C) 2020 Gregory Nutt. All rights reserved.
```

Comment:
need change too

Reply:
@xiaoxiang781216 This one is based on s32k1xx_flexcan.h which is under the other (old) licence, so the code should retain the copyright. Or do I understand it wrong?

Reply:
> 
> 
> @xiaoxiang781216 This one is based on s32k1xx_flexcan.h which is under the other (old) licence, so the code should retain the copyright. Or do I understand it wrong?

Yes, but I am the author/coyright holder on that file.  I have submitte both and ICLA and an SGA granting all NuttX software to the ASF.  The license has not yet been changed on s32k1xx_flexcan.h but it should be.

I also grant you the right to change the header to Apache 2.0.  No new files may come into the repository with the old BSD license.  So if the header is not changed, the the PR will have to be declined.

Reply:
@patacongo Thank you for clarifying this. I´ll make the changes.